i18n/en/en.yaml

```diff
@@ -69,7 +69,7 @@ home_show_less:
 home_falco_what_is_falco_title:
   other: Detect security threats in real time
 home_falco_what_is_falco:
-  other: Falco is the open source solution for runtime security for hosts, containers, Kubernetes and the cloud. Get real-time visibility into abnormal behaviors and potential security threats, intrusions, and data theft or compliance violations.
+  other: Falco, a cloud-native security tool for Linux, employs custom rules on kernel events, enriched with container and Kubernetes metadata, to alert in real time. Enhance your off-host analysis and gain visibility into abnormal behavior, security threats, and compliance violations for comprehensive runtime security.
```

Comment:
```suggestion
  other: Falco, a cloud-native security tool, employs custom rules on system events, enriched with container and Kubernetes metadata, to alert in real time. Enhance your off-host analysis and gain visibility into abnormal behavior, security threats, and compliance violations for comprehensive runtime security.
```
WDYT?

Reply:
Why take away Linux and kernel mentions?

Reply:
I agree with @nate-double-u. We should keep them.